### ISSUE TITLE
How do I access buttons on a dialog window in headless chrome?

### ISSUE THREAD

**Issue by leonardstock** [SAP/ui5-uiveri5#110]
Hi,
I switched from headful Chrome to headless and suddenly I cannot access any buttons on dialog popover windows due to them not being found, which worked just fine in headful mode. How do I access those in headless mode? 
Further information and my code is linked in my question on stackoverflow: https://stackoverflow.com/questions/57623451/how-to-interact-with-a-dialog-window-in-chrome-headless-with-uiveri5

**Comment by maximnaidenov** [comment]
Hi Leon,
Can you please attach the screenshot for the respective step from headed and headless execution ?.

**Comment by leonardstock** [comment]
Thank you for the tip with the screenshots, it helped a lot! Just to clarify, I wrote the tests in German and therefore referred to the text on the button as "Nein", which means no. When I reviewed the screenshots, I realized that headless chrome automatically translates to English, which headful chrome does not. Closing this issue as my problem is therefore solved if I build my tests in English.
